engine/src/main/scala/cromwell/engine/backend/JobDescriptor.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Try
   *
   * @tparam K CallKey subtype
   */
-sealed trait JobDescriptor[K <: JobKey] {
+sealed trait JobDescriptor[K <: CallKey] {
```

Comment:
I just checked the "develop" branch.  It is very cool.  I will give it a try to put the LSF as a supportedBackends.